litellm/integrations/langfuse.py

```diff
@@ -299,11 +301,24 @@ def _log_langfuse_v2(
                     else:
                         clean_metadata[key] = value
 
+            api_base = litellm_params.get("api_base", None)
```

Comment:
@Manouchehri is this trying to do something similar to this function - https://github.com/BerriAI/litellm/blob/ef4c85522c001c930f02e2ec2c32dea9a7816b74/litellm/utils.py#L5319

Reply:
merging this for now and will refactor to to use the `get_api_base`